code/modules/surgery/organs/lungs.dm

```diff
@@ -82,13 +83,13 @@
 /obj/item/organ/lungs/proc/check_breath(datum/gas_mixture/breath, mob/living/carbon/human/H)
 //TODO: add lung damage = less oxygen gains
 	var/breathModifier = (5-(5*(damage/maxHealth)/2)) //range 2.5 - 5
-	if(H.status_flags & GODMODE)
+	if((H.status_flags & GODMODE))
```

Comment:
does this second pair do something special?

Reply:
it makes traumatized coders feel safe